src/components/TabbedFilter/TabbedFilter.jsx

```diff
@@ -1,26 +1,46 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import { filterMovies } from 'store/actions';
 import PropTypes from 'prop-types';
 import styles from './TabbedFilter.styl';
 import TabItem from '../TabItem/TabItem';
 
-const TabbedFilter = ({ items = [] }) => {
+const TabbedFilter = ({ items = [], filterMovies }) => {
   if (!items?.length) return null;
 
+  items.map((item, i) => ({...item.id = i}));
```

Comment:
waaaaaaaaait a second 😱 Have you just mutated props?

Reply:
Haha, looks like that. Yes, it's a mistake